src/rust/en.nana/res/filters.json

```
@@ -0,0 +1,5 @@
+[
+	{ 
+		"type": "title" 
+	}
+]
```

still needs a newline at the end of the file
```suggestion
]

```